test/test_gmm.py

```diff
@@ -374,6 +374,81 @@ def test_gmm_backward(self):
     # Make sure gmm doesn't fallback.
     self.assertNotIn("aten::", met.short_metrics_report())
 
+  @unittest.skipIf(xr.device_type() != 'TPU', "This test only works on TPU.")
```

Comment:
do you need TPU version check here?

Reply:
lol, good question.

Reply:
v2 is pretty happy on the tree.

Reply:
interesting.. I thought pallas is not supported on v2.

Reply:
gmm is just mm... The kernel is simple... Other fuses softmax, etc...